src/model/mod.rs

```diff
@@ -18,7 +18,7 @@ use model::sample::{Sample, Observation};
 
 pub type AlleleFreq = NotNaN<f64>;
 pub type DiscreteAlleleFreqs = Vec<AlleleFreq>;
-pub type ContinuousAlleleFreqs = Range<AlleleFreq>;
+pub type ContinuousAlleleFreqs = AFRange;
```

Comment:
I think we have redundant type names now. Maybe simply call `AFRange` `ContinuousAlleleFreqs` instead?